util/docker.go

```diff
@@ -336,7 +336,7 @@ func DockerPull(image, registry, org, username, password string) (string, error)
 		registry = constants.DefaultRegistry
 	}
 
-	registry = strings.Replace(registry, "https://hub.docker.com", "docker.io", 1)
+	registry = strings.Replace(registry, "https://hub.docker.com/", "docker.io", 1)
 
```

Comment:
We want to match the default registry constant so this is good, although we might need to check elsewhere and make sure we're adding the trailing slash (i.e. I think that's why tests are failing now?)

Reply:
Alright, yeah there were like 4 places where I though this action could occurs.  I will look at the failures.